backend/src/v4/routes/model.js

```diff
@@ -722,7 +722,7 @@ router.put("/:model", middlewares.hasEditAccessToFedModel, middlewares.formatV5N
  *    }
  * ]
  */
-router.post("/models/permissions", middlewares.hasEditPermissionsAccessToMulitpleModels, updateMultiplePermissions);
```

Comment:
does this make sense? If i'm reading you correctly, you're making this POST request do the same thing as the PATCH request.

Which shouldn't be the case? POST should overwrite the array where as PATCH should make changes to it?